### DISCUSSION TITLE
What do you think should be added/changed/removed?

### DISCUSSION THREAD

**Discussion by svbutko** [svbutko/react-native-template-strong#31] [Ideas]
Maybe there are some dependencies you'd like to see here or documentation is not describing some of the cases?

There are some issues which I created and would like to add in next versions you can check them [here](https://github.com/svbutko/react-native-template-strong/milestone/1)

Every feedback is appreciated!

**Comment by JB712** [comment]
Hi ! thanks for this project !
When I scroll all the packages I can search for each package it's usage but I wonder why you think it is important and if I need it.
I think you could benefit to have a page in your doc that describes why you did include (or not) every package that is in your project (like you partially did in FAQ) 😃

**Reply by svbutko** [reply]
Hi, thank you for suggestion, I'll add it soon!

**Comment by kentdev92** [comment]
Firstly i have to thank you so much for your working and i really like it. But can you add more document that guide us to change redux thunk to redux saga, because some complicated business logic i think we need saga or new listener feature of rtk than redux thunk. Please consider my request. Thank you so much and have a good day! @svbutko

**Reply by svbutko** [reply]
Hello, thank you for your feedback!
I can try to add a documentation regarding it with some limited explanations of what should be done in order to switch

**Reply by kentdev92** [reply]
@svbutko Thank you so much for your kind. I will be waiting for you.
